src/Blocks/BlockPressurePlate.h

```diff
@@ -28,7 +28,26 @@ class cBlockPressurePlateHandler :
 			return false;
 		}
 
-		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)));
+		// TODO: check if the block is upside-down slab or upside-down stairs
+		BLOCKTYPE Block = a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ);
```

Comment:
Thies needs a check (or a change to the check above) - if `a_RelY` is zero, this will crash.